webapp/loggedin/tableViews/tableView.js

```diff
@@ -90,8 +93,8 @@ TableView.defaultProps = {
   headerLeftComponent: () => <div></div>,
   rowHeaderComponent: () => <div></div>,
   rowComponent: () => <div></div>,
-  noItemsLabelKey: 'common.noItems'
-
+  noItemsLabelKey: 'common.noItems',
+  onRowClick: () => {},
```

Comment:
it should be null by default, check if it's not null on click

Reply:
ok